core-domain/src/main/java/org/uniprot/core/publication/impl/MappedPublicationsImpl.java

```diff
@@ -0,0 +1,65 @@
+package org.uniprot.core.publication.impl;
+
+import org.uniprot.core.publication.CommunityMappedReference;
+import org.uniprot.core.publication.ComputationallyMappedReference;
+import org.uniprot.core.publication.MappedPublications;
+import org.uniprot.core.publication.UniProtKBMappedReference;
+import org.uniprot.core.util.Utils;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author sahmad
+ * @created 18/12/2020
+ */
+public class MappedPublicationsImpl implements MappedPublications {
+    private static final long serialVersionUID = -6303466445944690517L;
+    private final UniProtKBMappedReference uniProtKBMappedReference;
+    private final List<ComputationallyMappedReference> computationalMappedReferences;
+    private final List<CommunityMappedReference> communityMappedReferences;
```

Comment:
we should rename computationalMappedReferences to computationallyMappedReferences